src/main.rs

```diff
@@ -87,6 +87,14 @@ async fn main() -> Result<(), Error> {
             .map(ToString::to_string)
             .unwrap_or_else(|| pr.url.clone());
 
+        if let Some(labels) = &pr.labels {
+            let has_a_review = labels.iter().any(|label| label.name == "a-review");
+            if has_a_review {
```

Comment:
```suggestion
            let has_a_review = labels.iter().any(|label| label.name == "a-review");
            let has_e_review = labels.iter().any(|label| label.name == "e-review");
            if has_a_review && !has_e_review {
```

How about this? I think it's safer than just `if !has_e_review { ... }` in case the label bot breaks again.